CHANGELOG.md

```diff
@@ -1,3 +1,104 @@
+
+## New features
+
+A new BaseTextArea component that leverages UITextView to provide multi-line text input have been created at components/TextControls/src/BaseTextAreas/.
+
+## Component changes
+
+### ActionSheet
+
+* [Migrates Action sheet to MDCAvailability. (#9733)](https://github.com/material-components/material-components-ios/commit/652077c779b06fc6b44e9bbae237f47e083c053c) (Bryan Oltman)
+
+### ActivityIndicator
+
+* [Migrates activity indicator to MDCAvailability. (#9735)](https://github.com/material-components/material-components-ios/commit/28a7fac63f7ee5f1787ab56190c8a579532810d6) (Bryan Oltman)
+
+### BottomNavigation
+
+* [Migrates Bottom Navigation to MDCAvailability. (#9717)](https://github.com/material-components/material-components-ios/commit/02efdd188990305ff298fbad25fa48634974b1b4) (Randall Li)
+
+### Buttons
+
+* [Fix tvOS target bug. (#9705)](https://github.com/material-components/material-components-ios/commit/95518f48d4fe6c10e13cbdcfcc3a1c863697298d) (Randall Li)
+* [MDCButton. (#9667)](https://github.com/material-components/material-components-ios/commit/1d8d647f1f534de4366c8722992b67858d7ba444) (copybara-service[bot])
+* [Delete deprecated MDCOutlinedButtonThemer (#9688)](https://github.com/material-components/material-components-ios/commit/1118f14124858fe9a258b0971393e33b3966d2bf) (Bryan Oltman)
+
+### Cards
+
+* [Migrates cards to MDCAvailability. (#9728)](https://github.com/material-components/material-components-ios/commit/b677a3629e180a89f1e6a15256b45790ca794af5) (Randall Li)
+
+### Chips
+
+* [Migrates Chips to MDCAvailability. (#9713)](https://github.com/material-components/material-components-ios/commit/33fbdfc3be62b4ff78951898b9abe6fc59041db4) (Randall Li)
+
+### Dialogs
+
+* [Add snapshot tests for additional dialog configurations. (#9646)](https://github.com/material-components/material-components-ios/commit/e2bff768df18fc24f91857447ddb8992071e778f) (copybara-service[bot])
+* [Adding accessory view vertical inset to private header (#9692)](https://github.com/material-components/material-components-ios/commit/ecddca8d58d8e02fb350d31277693dbaff56f952) (Galia Kaufman)
+* [Adding an adjustable insets feature flag. Flag default is NO. Clients must enable flag if they want to customize the inset. This flag has no effect yet in this CL. Adjustable layout logic will be introduced in followup CLs. (#9648)](https://github.com/material-components/material-components-ios/commit/0c8647cfca40fc04aa9da1ff7dde30efbe6456dc) (copybara-service[bot])
+* [Delete Dialogs OWNERS file (#9697)](https://github.com/material-components/material-components-ios/commit/5884725f8746bfaecbc02ed509cb70eac42c5428) (Randall Li)
+* [Fix custom title icon view layout - top inset (#9721)](https://github.com/material-components/material-components-ios/commit/3b89aecee05c95d49f331ce77427cc5731fbea99) (Galia Kaufman)
```

Comment:
Can you attribute this to (Iryna Berezan) instead?

Reply:
Done